recipe/build.sh

```diff
@@ -77,3 +77,7 @@ else
 fi
 
 make install
+# 2022/03/06: hmaarrfk
+# x265 likes to install the static library no matter what
+# Remove any installed static libraries
+rm ${PREFIX}/lib/*.a
```

Comment:
Are you sure this is correct? We just built a bunch of static libs above (line 70-76)

Reply:
hmm, i didn't read the `build.sh` file that carefully ilt seems.

Basically, the conda-forge stance is that static libraries should be shipped separately. 
https://github.com/conda-forge/cfep/blob/main/cfep-18.md

I was hoping nobody was relying on them, and that I could simply remove them.

if it were up to me, I would simply remove lines 66 through 77 and wait until somebody else complains about it.

I'm not sure if that somebody is you Matthias. I'm happy to "include" static packages for now.

My main motivation here is to build windows package.s

Reply:
Ultimately. from an end user's point of view, the package is 100% larger when we include the static library. 

Approximate numbers:
Download size is 6 MB vs 3, and extracted size is 50MB instead of 25MB on linux 64 bit x86

Therefore, as somebody who downloads packages many more times than I build them, I think it is very beneficial to remove the static libraries.

Reply:
Yeah, I think you are right. `bin/x265` seems to use the shared library already, so let's remove the static one unless someone complains. Please go ahead and remove lines 66-77.